examples/eks_managed_node_group/main.tf

```diff
@@ -568,11 +568,6 @@ resource "aws_launch_template" "external" {
     enabled = true
   }
 
-  network_interfaces {
```

Comment:
I'm not sure why this example started complaining about setting a device index so just removing since its not required for the example

Reply:
There was a bug in the AWS provider about it - https://github.com/hashicorp/terraform-provider-aws/issues/23766

We hit it in one of the projects but I don't remember the resolution (probably downgraded version). Maybe leave a comment around this section with the link to the issue, so we know what was it in the future?

Reply:
ah, good to know - thought I was losing it 😅 

Added back the block but commented it out with a link to the issue - https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1966/commits/bf45e14950475d3c527618f48a20c940ab621c1e